test/__mocks__/validation.mock.ts

```diff
@@ -83,6 +104,30 @@ export const BENEFICIAL_OWNER_INDIVIDUAL_WITH_MAX_LENGTH_FIELDS_MOCK = {
   ...START_DATE_MOCK
 };
 
+export const BENEFICIAL_OWNER_INDIVIDUAL_WITH_INVALID_CHARS_MOCK = {
+  first_name: FIRST_NAME_INVALID_CHARS,
+  last_name: NAME_INVALID_CHARS,
+  nationality: INVALID_NATIONALITY,
+  is_on_sanctions_list: "0",
+  is_service_address_same_as_usual_residential_address: "1",
+  ...RESIDENTIAL_ADDRESS_WITH_INVALID_CHAR_FIELDS_MOCK,
+  ...SERVICE_ADDRESS_WITH_INVALID_CHAR_FIELDS_MOCK,
+  ...DATE_OF_BIRTH_MOCK,
+  ...START_DATE_MOCK
+};
+
+export const BENEFICIAL_OWNER_INDIVIDUAL_WITH_INVALID_CHARS_SERVICE_ADDRESS_MOCK = {
+  first_name: MAX_50 + "1",
```

Comment:
Do you really want to exceed max length for these fields? This data structure is concerned with testing invalid chars (and only in the service address).

Reply:
No this should be changed.